grafana/dashboards/battery-health.json

```diff
@@ -1458,13 +1512,13 @@
           "type": "postgres",
           "uid": "TeslaMate"
         },
-        "definition": "-- CONCATENATED JOIN QUERIES TO IMPROVE PERFORMANCE\n-- The following query is the result of many tests and hours of work. This panel is for your own personal use. \n-- If you think you can improve it and contribute, please create a pull request and do not take it to your repository, \n-- much less upload it to another repository as if the original idea were yours, nor do you share it on social media\n-- without mentioning the author. Respect the ingenuity and work of others. Cheers!\n-- 2023-07-21\n-- By @jheredianet - Twitter: @juanheredia",
+        "definition": "-- CONCATENATED JOIN QUERIES TO IMPROVE PERFORMANCE\n-- The following query is the result of many tests and hours of work. This panel is for your own personal use. \n-- If you think you can improve it and contribute, please create a pull request and do not take it to your repository, \n-- much less upload it to another repository as if the original idea were yours, nor do you share it on social media\n-- without mentioning the author. Respect the ingenuity and work of others. Cheers!\n-- 2024-01-06\n-- By @jheredianet - Twitter: @juanheredia",

```

Comment:
The name and Twitter mention is not new, but I am not sure how I feel about it. So in the end it is a no for me and I also see it in conflict with the MIT license of TeslaMate.
What do you think @DrMichael , @cwanja , @brianmay , @Dulanic , @tobiasehlert ?

Reply:
Agree with the MIT license angle perspective. Not sure anyone would come "after" us, but in good faith we should probably remove this line for this repo specifically. Happy to give credit in the docs?

Reply:
I agree with @cwanja, creds are given in the changelog if you ask me.

I've never worked in a single repo or project where you write such things inside of the code. That would additionally become very messy considering over 150 contributors just in TeslaMate. Even though Adrian has approved PR #3059 earlier with such thing, I still don't thinks it's supposed to be there.

Reply:
Hi, if you feel confortable, I may do other commit an remove my Twitter name. Have in mind that this is a "code comment" which is not seen anywhere on the Grafana dashboard, it's just a note for those who have the necessary knowledge to analyze the code and know who the author is in case they have any technical questions.
As I said, if you prefer, let me know what you decide and I'll do the corresponding change.

Reply:
@jheredianet Thanks again for your change. Please remove your name mention. Your work will be visible as named contributor in changelog and in git history.

Reply:
Done! Thanks.
Cheers!!!